clang/test/CodeGenSYCL/integration_header.cpp

```diff
@@ -121,5 +149,11 @@ int main() {
       }
   });
 
+  // FIXME: We cannot use the member-capture because non-integration headers
+  // don't handle these types right.
```

Comment:
@erichkeane , did you mean that all handlers except integration header handlers don't handle base types?

Reply:
I think so, basically the accessor/stream classes in bases aren't handled elsewhere, so we end up getting a crash if we do this with -c. (Which is why the test changed it to syntax-only).

Reply:
Can I rewrite this comment a bit then?

Reply:
Of course!